doc/how-tos/removing_data_from_pachyderm.md

```diff
@@ -0,0 +1,135 @@
+
+- [Delete the HEAD of a Branch](#deleting-the-head-of-a-branch).
+If the incorrect data was added in the latest commit and no additional
+data was committed since then, follow the steps in this section to fix
+the HEAD of the corrupted branch.
+- [Delete non-HEAD Commits](#deleting-non-head-commits). If after
+committing the incorrect data, you have added more data to the same
+branch, follow the steps in this section to delete corrupted files.
+- [Deleting sensitive data](#deleting-sensitive-data). If the bad
+commit included sensitive data that you need immediately and completely
+erase from Pachyderm, follow the steps in this section to purge data.
+
+## Deleting the HEAD of a Branch
+
+If you have just committed incorrect, corrupt, or otherwise bad
+data to a branch in a Pachyderm repository, the HEAD of your branch,
+or the latest commit is bad. Users who read from that commit
+might be misled, and pipelines subscribed to it might fail or
+produce bad downstream output. You can solve this issue by running
+the `pachctl delete commit` command.
+
+To fix a broken HEAD, complete the following steps:
+
+```bash
+$ pachctl delete commit <repo>@<branch-or-commit-id>
+```
+
+When you delete a bad commit, Pachyderm performs the following actions:
+
+- Deletes the commit metadata.
+- Changes HEADs of all the branches that had the bad commit as their
+  HEAD to the bad commit parent. If the bad commit does not have
+  a parent, Pachyderm sets the branch's HEAD to `nil`.
+- If the bad commit has children, sets their parents to the deleted commit
+  parent. If the deleted commit does not have a parent, then the
+  children commit parents are set to `nil`.
+- Deletes all the jobs that were triggered by the bad commit. Also,
+  Pachyderm interrupts all running jobs, including not only the
+  jobs that use the bad commit as a direct input but also the ones farther
+  downstream in your DAG.
+- Deletes the output commits from the deleted jobs. All the actions
+  listed above are applied to those commits as well.
+
+## Delete Old Commits
+
+If you have committed more data to the branch after the bad data
+was added, you can try to delete the commit as described in
+[Deleting the HEAD of a Branch](#deleting-the-head-of-a-branch).
+However, unless the subsequent commits overwrote or deleted the
+bad data, the bad data might still be present in the
+children commits. Deleting a commit does not modify its children.
+
+In Git terms, `pachctl delete commit` is equivalent to squashing a
+commit out of existence, such as with the `git reset --hard` command.
+The `delete commit` command is not equivalent to reverting a
+commit in Git. The reason for this
+behavior is that the semantics of revert can get ambiguous
+when the files that are being reverted have been
+otherwise modified. Because Pachyderm is a centralized system
+and the volume of data that you typically store in Pachyderm is
+large, the concept of merge conflicts that you might run into when
```

Comment:
> ...  the concept of merge conflicts that you might run into when
you work with Git is irrelevant.

this sentence in confusing. change to:

...large, merge conflicts can quickly become untenable so Pachyderm prevents merge conflicts entirely.